Workflow/Sources/Workflow/Models/AnyWorkflow.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  AnyWorkflow.swift
+//  
+//
+//  Created by Tyler Thompson on 11/25/20.
+//
+
+import Foundation
+
+public class AnyWorkflow: LinkedList<FlowRepresentableMetaData> {
+    public typealias InstanceNode = LinkedList<AnyFlowRepresentable?>.Element
+    public typealias ArrayLiteralElement = AnyFlowRepresentable.Type
+    internal var instances = LinkedList<AnyFlowRepresentable?>()
+    internal var orchestrationResponder: AnyOrchestrationResponder?
+
+    public var firstLoadedInstance: LinkedList<AnyFlowRepresentable?>.Element?
+
+    public func applyOrchestrationResponder(_ orchestrationResponder: AnyOrchestrationResponder) {
+        self.orchestrationResponder = orchestrationResponder
+    }
+
+    @discardableResult public func launch(withLaunchStyle launchStyle: LaunchStyle = .default,
+                                          onFinish: ((Any?) -> Void)? = nil) -> LinkedList<AnyFlowRepresentable?>.Element? {
+        launch(passedArgs: .none, withLaunchStyle: launchStyle, onFinish: onFinish)
+    }
+
+    @discardableResult public func launch(with args: Any?,
+                                          withLaunchStyle launchStyle: LaunchStyle = .default,
+                                          onFinish: ((Any?) -> Void)? = nil) -> LinkedList<AnyFlowRepresentable?>.Element? {
+        launch(passedArgs: .args(args), withLaunchStyle: launchStyle, onFinish: onFinish)
+    }
+
+    @discardableResult public func launch(passedArgs: PassedArgs,
+                                          withLaunchStyle launchStyle: LaunchStyle = .default,
+                                          onFinish: ((Any?) -> Void)? = nil) -> LinkedList<AnyFlowRepresentable?>.Element? {
+        removeInstances()
+        instances = LinkedList(map { _ in nil })
+        var root: (instance: AnyFlowRepresentable, metadata: FlowRepresentableMetaData)?
+        var passedArgs = passedArgs
+
+        let metadata = first?.traverse { [self] nextNode in
+            let nextMetadata = nextNode.value
+            let flowRepresentable = nextMetadata.flowRepresentableFactory()
+            flowRepresentable.workflow = self
+            flowRepresentable.proceedInWorkflowStorage = { passedArgs = $0 }
+
+            let shouldLoad = flowRepresentable.shouldLoad(with: passedArgs)
+
+            defer {
+                guard let instance = instances.first?.traverse(nextNode.position) else { fatalError("Internal state of workflow completely mangled somehow...") }
+                let persistance = nextMetadata.calculatePersistance(passedArgs)
```

Comment:
Leaving a note here so we don't forget.  We need to fix the spelling of 
`Persistance` to `Persistence`
```suggestion
                let persistence = nextMetadata.calculatePersistence(passedArgs)
```
everywhere.  This is a big change that would break the API so we should do it in this version ramp instead of making 2.